scripts/moveFiles.bash

```diff
@@ -6,6 +6,12 @@ pushd ../publicgoods-website/registry && \
     git add static/* && \
     cp ../../publicgoods-scripts/packages/registry/build/index.html . && \
 popd
+pushd packages/eligibility && \
+  ln -s ../../../../publicgoods-website/wp-includes/
+  ln -s ../../../../publicgoods-website/wp-content/
+  npm install && npm run build && \
```

Comment:
@nathanfletcher If you remove this line, then this PR looks will look good to me.